doc/stdlib/linalg.rst

```diff
@@ -101,6 +101,10 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Compute a factorization of a positive definite matrix ``A`` such that ``A=L*Diagonal(d)*L'`` where ``L`` is a unit lower triangular matrix and ``d`` is a vector with non-negative elements.
 
+.. function:: ldltfact(A) -> CHOLMOD.Factor
+
+   Compute the LDLt factorization of a sparse symmetric or Hermitian matrix ``A``. A fill-reducing permutation is used.  The main application of this type is to solve systems of equations with ``\``, but also the methods ``diag``, ``det``, ``logdet`` are defined. The function calls the C library CHOLMOD and many other functions from the library are wrapped but not exported.
```

Comment:
Looks like deada3e35ed50b2afadca2f9042368ecc42ec484 helps, but we need more than "a line or two." The documentation should be as explicit as possible. When we say ldlt, we need to specify exactly what we mean by that. `D` is a diagonal matrix and this factorization may fail for indefinite matrices (there are of course important special cases where it won't fail, but that's not straightforward to detect in general).

Arguably the names we're using are better choices than Matlab's, but we don't exist in a vacuum, there's a lot of precedent and we should be especially clear in places where we differ from that precedent.

Splitting these up to give separate access to different factorization types is good, though maybe we could eventually do it with dispatch on a `Val{:Cholesky}` or similar, instead of having so many different function names.